bin/post_compile

```diff
@@ -4,15 +4,21 @@
 # Make non-zero exit codes & other errors fatal.
 set -euo pipefail
 
-# Create a `prod/` directory containing built/minified versions of the frontend assets.
+# Create a `build/` directory containing built/minified versions of the frontend assets.
 # Uses the node binaries/packages installed by the nodejs buildpack previously.
 npm install && npm run build
 
+# Pull in the Pyodide static resources from a Github release
+export PYODIDE_VERSION=0.1.10
```

Comment:
I wonder, should we default to getting the latest release on github? A little more involved, but would save us from having to constantly update this file whenever a new pyodide release goes out.

https://developer.github.com/v3/repos/releases/

We should probably also let the admin select a specific release by specifying a PYODIDE_VERSION environment variable.

Reply:
I actually think we should be explicit about the version here, or at least pin to a major semver or something.  There have been problems with mismatching iodide and pyodide in the past (like when changes to the local variables pane broke things, or the recent `asyncEvaluator` changes).  Ideally, we should probably do some sort of integration test...

Reply:
I think it does! Having pyodide easily available for the local dev use case would be a huge win.